dhis-2/dhis-services/dhis-service-reporting/src/test/java/org/hisp/dhis/interpretation/InterpretationServiceTest.java

```diff
@@ -34,6 +34,7 @@
 import org.hisp.dhis.chart.ChartService;
```

Comment:
I can run tests of any other module, but not dhis-services/dhis-service-reporting/, I think I am missing something some setup:

```
$ mvn test -f dhis-services/dhis-service-reporting/ 
* ERROR 17:23:12,861 Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@749aa36f] to prepare test instance [org.hisp.dhis.datastatistics.DataStatisticsEventStoreTest@5f966dee] (TestContextManager.java [main])
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124)
[...]
```

Reply:
Not sure about that way of running the tests. This is [what I found](https://www.dhis2.org/development#maven) and this is the output I am getting for the Interpretation tests...
![screenshot from 2018-01-07 16-02-35](https://user-images.githubusercontent.com/6850223/34651148-367fa544-f3c4-11e7-8194-b620c2111703.png)